### PR TITLE
creating the template for the login page

### PR DESCRIPTION
**Pull request by Leandro-Custodio**
:rage: **PROBLEMA**
you need to create a new login

:sweat_smile: **SOLUÇÃO**
 creating the template for the login page

[comment]: <> (❗❗ IMPORTANTE ❗❗)

:bug: **TESTES**
It was not necessary to develop tests for this correction.

:heavy_check_mark: **ATIVIDADE**
https://trello.com/c/jqQiVUXq/10-desenvolver-o-front-end

@Leandro-Custodio

### CODE REVIEW

File: src/pages/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+// import '../styles/pages/landing.css';
+
+function Login() {
+  return (
+    <div id="page-landing">
+      <div className="content-wrapper">
+        <main>
+          <h1>Leve felicidade para o mundo</h1>
```
Comment (GLamim) [inline src/pages/Login.tsx]:
```suggestion
          <h1>Bem Vindo ao HerePresent</h1>
```

Comment (lucas-besen) [inline src/pages/Login.tsx]:
```suggestion
          <h1>Evoluindo o seu modo de fazer chamada</h1>
```

**Review by lucas-besen — CHANGES_REQUESTED** [review]

File: src/pages/Login.tsx (hunk 1 above)
Comment (Leandro-Custodio) [reply src/pages/Login.tsx]: blz show

**Review by GLamim — APPROVED** [review]
Tudo certinho

**Review by lucas-besen — APPROVED** [review]

**Pull request merged**